test/membrane/aac/parser_test.exs

```diff
@@ -76,7 +76,9 @@ defmodule Membrane.AAC.ParserTest do
         # Channel configuration - stereo
         2::4,
         # frame length - 960 samples
-        0b100
+        1::1,
+        0::1,
+        0::1
```

Comment:
If that's supposed to be one number coded on 3 bits then I'd rather do:
```suggestion
        0b100::3
```

Reply:
This is not a single number, these are 3 separate flags, forming the structure called GASpecificConfig:
- frameLengthFlag
- dependsOnCoreCoder
- extensionFlag
I think this is a little more readable this way, but I will leave some comments explaining what these mean